### PR TITLE
Add in class instructions

### PR DESCRIPTION
**Pull request by cht8687**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,52 @@
...
+
+Each class has average length of 2 hours. Each week, we will pick a time and place and announce it.
+
+
+
+## Materials that we used for this class
+
+
+
+Currently we are using [Yorgey's cis194 course](https://github.com/bitemyapp/learnhaskell#yorgeys-cis194-course).
```
Comment (stevemao) [inline README.md]: There are a few different versions. We should link which version we are using

File: README.md [hunk 2]
```diff
@@ -1,2 +1,52 @@
 # 2019-winter-Haskell-school
+
 2019 winter Haskell school in Sydney Australia 
+
+
+
+## Class timetables and planning
+
+This winter session is initially designed to have 10 classes(might extended).
```
Comment (shineli1984) [inline README.md]: CIS194 is a 12 week course and we will extend it with advanced contents.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,52 @@
...
+
+This course is widely recommended by Haskell learners. 
+
+
+
+## homework and after class discussions
+
+
+
+After each week's class, participants can complete assignments from the cis194 course and submit to this repo as a [Pull Request](https://github.com/fp-works/2019-winter-Haskell-school/pulls).
```
Comment (shineli1984) [inline README.md]: Can we say `need to complete` to emphasis on the importance of homework? Or potentially a section to point out writing Haskell is the only way of learning Haskell.

File: README.md (hunk 1 above)
Comment (cht8687) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (cht8687) [reply README.md]: updated.

File: README.md (hunk 3 above)
Comment (cht8687) [reply README.md]: updated. 
The session to point out writing Haskell can be in a separate PR.
Could you help? I am not expert in Haskell. 😄

Comment (shineli1984) [reply README.md]: I meant something like `if you don't do your homework and expect to be a Haskell programmer in the end of this course just by coming/reading the lecture, you are dreaming.`. But you know, in a nicer way.

Comment (cht8687) [reply README.md]: done. 😄

**Pull request merged**